lib/TPP/ToBlockLayoutAndBack.cpp

```diff
@@ -650,8 +652,9 @@ struct PropagateThroughPadOp : public OpRewritePattern<tensor::PadOp> {
 // %1 = linalg.fill ins(%cst) outs(%0)
 // %2 = tensor.pack %1 into %packed
 // %3 = some_packed_op %2
-// %4 = tensor.unpack %3 into %1
+//
```

Comment:
In the test, the tensor is still unpacking into the empty `%0`, so the comment should probably reflect that